src/org/jitsi/impl/neomedia/recording/SynchronizerImpl.java

```diff
@@ -457,18 +449,15 @@ private int getPacketType(RawPacket pkt)
      */
     void removeMapping(long ssrc)
     {
-        if (ssrcs.containsKey(ssrc))
+        synchronized (ssrcs)
         {
-            synchronized (ssrcs)
+            SSRCDesc ssrcDesc = ssrcs.get(ssrc);
```

Comment:
You are right, the read doesn't need to be protected here. I simply refactored the existing code. Will remove it.